cve_bin_tool/checkers/commons_compress.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for Apache commons-compress:
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-45/product_id-59066/Apache-Commons-Compress.html
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class CommonsCompressChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"Apache Commons Compress software defines an API for working with",
+        r"<url>http://commons.apache.org/proper/commons-compress/</url>",
+    ]
+    FILENAME_PATTERNS = [r"commons-compress(-[0-9]+\.[0-9]+(\.[0-9]+)?)?.jar"]
+    VERSION_PATTERNS = [
+        r"<artifactId>commons-compress</artifactId>\n  <version>([0-9]+\.[0-9]+(\.[0-9]+)?)</version>"
```

Comment:
Ah, the problem might be here.  in windows returns are usually \n\r so this might work:
```suggestion
        r"<artifactId>commons-compress</artifactId>\n\r?  <version>([0-9]+\.[0-9]+(\.[0-9]+)?)</version>"
```

Reply:
thanks